stream_alert_cli/terraform/handlers.py

```diff
@@ -205,16 +181,19 @@ def _terraform_destroy(options, config):
         return
 
     # Remove old Terraform files
-    _terraform_clean(config)
+    terraform_clean_handler(config, prompt=False)
 
 
-def _terraform_clean(config):
+def terraform_clean_handler(config, prompt=True):
     """Remove leftover Terraform statefiles and main/cluster files
 
     Args:
         config (CLIConfig): Loaded StreamAlert CLI
     """
-    LOGGER_CLI.info('Cleaning Terraform files')
+    if prompt and not continue_prompt(message='Are you sure you want to remove Terraform files?'):
```

Comment:
Why do we need to confirm the deletion of terraform json files? It doesn't change situation that the infrastructure has been destroyed even if user input `No` when we ask.

Reply:
you're probably right.. this is a hold over that can probably go away :)